assets/src/components/animation-controls.js

```diff
@@ -28,12 +35,16 @@ function AnimationControls( { setAttributes, attributes, animatedBlocks } ) {
 		},
 	];
 
-	animatedBlocks.map( ( block ) => {
+	getAnimatedBlocks().map( ( block ) => {
```

Comment:
Right now, content changes aren't reflected in the dropdowns. So when I edit one text block, its excerpt should change in the dropdown of the other text block when editing animation settings.

@miina Perhaps you can see why this happens? I feel like I don't see the forest for the trees right now.

Reply:
Looks like it's just so that Gutenberg doesn't update the state immediately.

When I type something in block A, and then switch to another block, using `wp.data.select( 'core/editor' ).getBlock(…)` in the console still shows the old content for block A.